geometry_msgs/PoseStamped end_position
---
bool success
int8 error_code

#error codes for put down
int8 NO_ERROR = 0
int8 HOLD_NO_BLOCK = 1      # arm already picked up a block, e.g. pickup has been called twice
int8 PLAN_PRE_RELEASE = 2   # no motion plan found for pre-release pose
int8 MOVEIT_SCENE_ERROR = 3 # moveit planning scene was not able to remove or add a block from or to scene
int8 PLAN_RELEASE = 4       # no motion plan found for release pose
int8 PLAN_POST_RELEASE = 5  # no motion plan found for release-grasp pose